plugins/dev-tools/src/field_test_helpers.mocha.js

```diff
@@ -86,6 +91,30 @@ function runCreationTests_(testCases, assertion, creation) {
   runTestCases(testCases, createTestFn);
 }
 
+/**
+ * Runs provided creation test cases.
+ * @param {Array<FieldCreationTestCase>} testCases The test cases to run.
+ * @param {function(new:Blockly.Field,FieldCreationTestCase):Blockly.Field
+ *    } creation A function that returns an instance of the field based on the
+ *    provided test case.
+ * @private
+ */
+function runCreationTestsAssertThrows_(testCases, creation) {
```

Comment:
Alternatively, I have considered adding a property on the test case. However, the only benefit I see there is that we would be able to test a field where some values throw and some revert to default, which I think seems unlikely (I would expect it to be one or the other, not both).